build/admin.mk

```diff
@@ -0,0 +1,33 @@
+
+OUTPUT_DIR  ?= ./output/
+LATEST_VER  ?= $(shell curl -s https://api.github.com/repos/newrelic/newrelic-cli/releases | jq -r '.[].tag_name' | head -n 1)
+
+admin: admin-init admin-homebrew
+
+admin-clean:
+	@echo "=== $(PROJECT_NAME) === [ admin-clean     ]: cleaning admin cache..."
+	@rm -rfv $(OUTPUT_DIR)/*
+
+admin-init:
+	@echo "=== $(PROJECT_NAME) === [ admin-init     ]: cloning homebrew..."
+	@test -d $(OUTPUT_DIR)/homebrew || \
+		git clone --depth 10 git@github.com:newrelic-forks/homebrew-core.git $(OUTPUT_DIR)/homebrew
+
+admin-homebrew:
+	@echo "=== $(PROJECT_NAME) === [ admin-homebrew ]: updating homebrew..."
+	@mkdir -p $(OUTPUT_DIR)/homebrew
+	@pushd $(OUTPUT_DIR)/homebrew \
+		; git checkout master \
+		; git fetch origin master \
+		; git reset --hard origin/master  \
+		; git checkout -b newrelic_$(LATEST_VER) \
+		; popd
+	@pushd $(OUTPUT_DIR)/homebrew; git checkout -b newrelic_$(LATEST_VER);popd
+	@sed -i '' -e 's/v[0-9]*\.[0-9]*\.[0-9]*/$(LATEST_VER)/g' \
+		output/homebrew/Formula/newrelic-cli.rb
```

Comment:
If you make references to the formula name a VAR, and scrape that out of the environment or a config file, we could port this back to newrelic/developer-toolkit-template-go repo

Reply:
Oh good idea.